### PR TITLE
GSoC blogs 2021

### PR DESCRIPTION
**Pull request by SunTzunami**
GSoC Blogs for weeks 8, 9, 10 and final work product.

### CODE REVIEW

File: docs/source/posts/2021/2021-08-02-week-9-sajag.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+   :author: Sajag Swami
+   :tags: google
+   :category: gsoc
+
+Welcome to the ninth weekly check-in. I'll be sharing my progress for the eighth week of coding.
+
+What did you do this week?
+--------------------------
+
+#. Updated `PR #452`_: Had an extra meeting with the mentors in which we fine-tuned the molecular module and optimised the code to make it more pythonic.
```
Comment (skoudoro) [inline docs/source/posts/2021/2021-08-02-week-9-sajag.rst:14]: weird rendering, to update

**Review by skoudoro — COMMENTED** [review]
Hi @SunTzunami,

Overall, it looks good. Some rendering are a bit weird so can you generate the doc locally and fix them? 

Thank you

File: docs/source/posts/2021/2021-08-23-final-work-sajag.rst [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+   required by `vtkProteinRibbonFilter` was initially unclear due to lack of
+   examples. I was able to understand what kind of output the filter required 
+   after a meeting with mentors. Tests were added and a demo was created.
+
+   *Pull Requests:*
+
+   -  **Ribbon representation:** https://github.com/fury-gl/fury/pull/452
+   -  **Ribbon Representation demo:** https://github.com/fury-gl/fury/pull/452
+
+- **Ball and Stick Representation**
```
[the remark this answers is not in the document]
Comment (SunTzunami) [reply docs/source/posts/2021/2021-08-23-final-work-sajag.rst]: Added empty lines

File: docs/source/posts/2021/2021-08-02-week-9-sajag.rst (hunk 1 above)
Comment (SunTzunami) [reply docs/source/posts/2021/2021-08-02-week-9-sajag.rst]: Updated

File: docs/source/posts/2021/2021-08-02-week-9-sajag.rst [hunk 3]
```diff
@@ -0,0 +1,44 @@
+Eighth week of coding!
```
[the remark this answers is not in the document]
Comment (SunTzunami) [reply docs/source/posts/2021/2021-08-02-week-9-sajag.rst]: Done

**Comment by SunTzunami** [comment]
Hi @skoudoro ! I updated the blogs and they look fine locally, PTAL.

**Comment by skoudoro** [comment]
Capture d’écran 2021-09-03 à 13 01 09


As you can see above, there are still some issues @SunTzunami . PTAL.

**Comment by SunTzunami** [comment]
Hi @skoudoro ! It looks fine locally: 

blog

**Comment by skoudoro** [comment]
How do you generate the docs? in your screenshot, can you please add the menu on the right. I will try to clean my repo and try to regenerate it from my side too

**Comment by SunTzunami** [comment]
Hi @skoudoro , I use this command to generate docs: `make -C . clean && make -C . html-no-examples`.  

I'm not sure about which menu did you mean (the menu is on the left for me).

Preview: 

blogs

**Comment by skoudoro** [comment]
Strange, I did exactly the same thing and I do not get the same rendering.

What is your sphinx version? (`pip show sphinx`)

me, I get this:
```
(fury-env-37) MacBook-Pro-de-koudoro:docs koudoro$ pip show sphinx
Name: Sphinx
Version: 2.2.0
Summary: Python documentation generator
Home-page: http://sphinx-doc.org/
Author: Georg Brandl
Author-email: georg@python.org
License: BSD
Location: /Users/koudoro/miniconda3/envs/fury-env-37/lib/python3.7/site-packages
Requires: packaging, sphinxcontrib-serializinghtml, Pygments, setuptools, sphinxcontrib-devhelp, sphinxcontrib-htmlhelp, snowballstemmer, sphinxcontrib-qthelp, Jinja2, sphinxcontrib-applehelp, alabaster, babel, requests, sphinxcontrib-jsmath, imagesize, docutils
Required-by: 
```

**Comment by SunTzunami** [comment]
```Name: Sphinx
Version: 4.1.2
Summary: Python documentation generator
Home-page: https://sphinx-doc.org/
Author: Georg Brandl
Author-email: georg@python.org
License: BSD
Location: /home/suntzunami/anaconda3/envs/env_fury/lib/python3.8/site-packages
Requires: sphinxcontrib-qthelp, sphinxcontrib-devhelp, docutils, sphinxcontrib-serializinghtml, Jinja2, imagesize, setuptools, sphinxcontrib-jsmath, packaging, requests, sphinxcontrib-applehelp, Pygments, snowballstemmer, babel, alabaster, sphinxcontrib-htmlhelp
Required-by: sphinx-rtd-theme, sphinx-gallery, sphinx-copybutton, ablog```

**Comment by skoudoro** [comment]
ok, strange, I updated my environment but I still can not get your result. I think it is from my side. 

I will go ahead and merge your PR. Thank you!

**Pull request merged**